main.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './school/actions/actions'
+
+class App
+  def initialize()
+    @actions = Actions.new
+  end
+
+  def init
+    main while @actions.keep_going
+  end
+
+  def do_action(option)
+    case option
+    when '1'
+      @actions.list_books
+    when '2'
+      @actions.list_people
+    when '3'
+      @actions.create_person
```

Comment:
- [ ] Creating a teacher prompts for `parent permission and `specialization` where it should ask only for the latter, please update it.